docs/en/config/_autotune.md

```diff
@@ -170,9 +170,19 @@ Fast oscillations (more than 1 oscillation per second): this is because the gain
 
 ### The auto-tuning sequence fails
 
+<div v-if="$frontmatter.frame === 'Multicopter'">
+
 If the drone was not moving enough during auto-tuning, the system identification algorithm might have issues to find the correct coefficients.
 
-Increase the <div style="display: inline;" v-if="$frontmatter.frame === 'Multicopter'">[MC_AT_SYSID_AMP](../advanced_config/parameter_reference.md#MC_AT_SYSID_AMP)</div><div style="display: inline;" v-else-if="$frontmatter.frame === 'Plane'">[FW_AT_SYSID_AMP](../advanced_config/parameter_reference.md#FW_AT_SYSID_AMP)</div> parameter by steps of 1 and trigger the auto-tune again.
+Increase the [MC_AT_SYSID_AMP](../advanced_config/parameter_reference.md#MC_AT_SYSID_AMP) parameter by steps of 1 and trigger the auto-tune again.
+
+</div>
+<div v-else-if="$frontmatter.frame === 'Plane'">
+
+By default, the autotune maneuvers ensure that a sufficient angular rate is reached for system identification. If the signal-to-noise ratio of the vehicle is low, the system identification algorithm might have issues finding the correct coefficients. Ensure that there is no excessive noise and/or platform vibration.
+
+</div>
+
```

Comment:
Worth specifying the expected rates here? A screenshot like you put into the PR description would be nice as well, with the amplitude increase noticeable.